tests/staticfiles_tests/test_storage.py

```diff
@@ -244,6 +244,56 @@ def test_cache_key_memcache_validation(self):
         self.assertEqual(cache_key, 'staticfiles:821ea71ef36f95b3922a77f7364670e7')
 
 
+@override_settings(
+    **dict(
+        TEST_SETTINGS,
+        STATICFILES_STORAGE=(
+            'staticfiles_tests.storage.MultiCachedStaticFilesStorage'
+        ),
+        DEBUG=False,  # see note above
+    )
+)
+class TestMultiCachedStorage(BaseCollectionTestCase, BaseStaticFilesTestCase,
+        SimpleTestCase):
+
+    def setUp(self):
+        storage.staticfiles_storage.hashed_files.clear()  # avoid cache interference
+        super(TestMultiCachedStorage, self).setUp()
+
+    def cached_file_path(self, path):
+        fullpath = self.render_template(self.static_template_snippet(path))
+        return fullpath.replace(settings.STATIC_URL, '')
+
+    def test_multi_extension_patterns(self):
+        """ See #19670 """
+        self.assertEqual(
+            settings.STATICFILES_STORAGE,
+            'staticfiles_tests.storage.MultiCachedStaticFilesStorage'
+        )
+        self.assertEqual(
```

Comment:
Not sure these asserts bring value ... they seem to test that `override_settings` works.